packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx

```diff
@@ -27,6 +27,7 @@ enum LabelValueTypes {
 
 const SUCCESSFULLY_COPIED_TEXT = 'Copied to clipboard';
 const SHOW_SUCCESS_DURATION = 2 * 1000;
+const HZ_PX_PER_CHAR = 7;
```

Comment:
what does `HZ` mean in this context? horizontal? might be worth spelling out here 😅

Reply:
`APPROXIMATE_HORIZONTAL_PIXELS_PER_CHARACTER`

Reply:
haha `HORIZONTAL_PX_PER_CHAR` is fine by me, I just think the `HZ` is a little confusing but the other abbreviations are pretty standard 😬